packages/js/src/components/modals/SEMrushUpsellAlert.js

```diff
@@ -19,7 +19,7 @@ const SEMrushUpsellAlert = () => {
 				sprintf(
 					/* translators: %s: Expands to "Yoast SEO". */
 					__(
-						"Would you like to be able to add these related keyphrases to the %s analysis so you can optimize your content even further?",
+						"You’ll reach more people with multiple keyphrases! Want to quickly add these related keyphrases to the %s analyses for even better content optimization?",
```

Comment:
Are we sure about the `´` here? I think we always use `'`

Reply:
I think we are not consistent with that. I copied this over from the document. And in general I think these tools tend to transform them to the "fancy" versions.